onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1529,6 +1529,19 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args) > 0,
         reason="this ATen overload only supports one tensor as input by design",
     ),
+    TorchLibOpInfo(
+        "multinomial",
+        core_ops.aten_multinomial,
+        nondeterministic=True,
+    ).xfail(
+        matcher=lambda sample: len(sample.input.shape) < 2,
+        reason="onnx spec requires input tensor to have at least 2 dimensions (the first one is batch size)",
```

Comment:
Can we use unqueeze and squeeze to handle this?

Reply:
Done